ui/imports/utils/ProfileUtils.qml

```diff
@@ -7,6 +7,7 @@ import StatusQ.Core.Theme 0.1
 QtObject {
 
     readonly property int defaultDelegateHeight: 76
```

Comment:
Unrelated to this PR, but I find it kind of strange to have a UI size defined in *Utils.qml. It should probably be defined in the ProfileShowcasePanel

Reply:
I found this a neutral place to add constants.. This value is used in different components of the `Profile` flow.. for example it's used also in the web tab panel that it doesn't know about the `ProfileShowcasePanel`. Any other idea?

Reply:
ohh, the web tab..I'll have a look since I'm working on the web tab 😄 
Thanks!